// @ali/dblx/lib/index.js
'use strict';

// this file is not used if use https://github.com/ant-design/babel-plugin-antd

var antd = {
  Alert: require('./alert/index'),
  //Rate: require('./rate/index'),
  Affix: require('./affix/index'),
  DatePicker: require('./date-picker/index'),
  Tooltip: require('./tooltip/index'),
  Carousel: require('./carousel/index'),
  Tabs: require('./tabs/index'),
  Modal: require('./modal/index'),
  Dropdown: require('./dropdown/index'),
  Progress: require('./progress/index'),
  Popover: require('./popover/index'),
  Select: require('./select/index'),
  Breadcrumb: require('./breadcrumb/index'),
  Popconfirm: require('./popconfirm/index'),
  Pagination: require('./pagination/index'),
  Steps: require('./steps'),
  InputNumber: require('./input-number/index'),
  Switch: require('./switch/index'),
  Checkbox: require('./checkbox/index'),
  Table: require('./table/index'),
  Tag: require('./tag/index'),
  Collapse: require('./collapse/index'),
  message: require('./message/index'),
  Slider: require('./slider/index'),
  QueueAnim: require('./queue-anim/index'),
  Radio: require('./radio/index'),
  notification: require('./notification/index'),

  Validation: require('./validation/index'),
  Tree: require('./tree/index'),
  TreeSelect: require('./tree-select/index'),
  Upload: require('./upload/index'),
  Badge: require('./badge/index'),
  Menu: require('./menu/index'),
  Timeline: require('./timeline/index'),
  Button: require('./button/index'),
  Icon: require('./icon/index'),
  Row: require('./row/index'),
  Col: require('./col/index'),
  Spin: require('./spin/index'),
  Form: require('./form/index'),
  Input: require('./input/index'),
  Calendar: require('./calendar/index'),
  TimePicker: require('./time-picker/index'),
  Transfer: require('./transfer/index'),
  Cascader: require('./cascader/index'),
  Spm: require('./Spm/index')
};

//Card: require('./card/index'),
//LocaleProvider: require('./locale-provider/index')
module.exports = antd;
